dwbb/src/interface/DW_stackctl.scala

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_stackctl
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import chisel3.util.log2Ceil
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    depth: Int = 16,
+    errMode: Boolean = false,
```

Comment:
I think we should define errMode with better documentation.

Reply:
Using `String`s instead of `Boolean`s in the revised code.